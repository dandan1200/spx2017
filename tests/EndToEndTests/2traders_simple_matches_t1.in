SELL 0 GPU 10 20;
