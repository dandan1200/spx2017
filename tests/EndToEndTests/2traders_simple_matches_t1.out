MARKET OPEN;
ACCEPTED 0;
FILL 0 10;
